server/services/employers.ts

```diff
@@ -198,7 +210,12 @@ export const getSiteByID = async (id: number): Promise<EmployerSite> => {
       'data.nonHcapOpportunity': 'true',
       'duplicateArchivedJoin.status': null,
     });
-  site[0].hcapHires = hcapHires;
-  site[0].nonHcapHires = nonHcapHires;
-  return site[0];
+
+  const siteResponse = {
+    ...site[0],
+    hcapHires,
+    nonHcapHires,
+    allocation: currentAllocation[0].allocation,
+  };
+  return siteResponse;
```

Comment:
No need to make an object and then return it. Simpler to just return directly.
```suggestion
  return {
    ...site[0],
    hcapHires,
    nonHcapHires,
    allocation: currentAllocation,
  };
```

Reply:
reverted back to the original return!